api/swagger.yaml

```diff
@@ -8339,6 +8339,29 @@ paths:
                   UsageData:
                     Size: 10920104
                     RefCount: 2
+              BuildCache:
+                -
+                  ID: "hw53o5aio51xtltp5xjp8v7fx",
```

Comment:
Arf, my bad; I didn't remove the trailing commas when I copy/pasta'd the example from JSON;

```
code.
[2021-07-08T21:22:49.541Z] api/swagger.yaml
[2021-07-08T21:22:49.541Z]   8344:50   error    syntax error: expected <block end>, but found ',' (syntax)
[2021-07-08T21:22:49.541Z] 
script returned exit code 1

Reply:
I should have double-checked before committing, next time I will do that!